staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation.go

```diff
@@ -128,6 +128,9 @@ type validationOptions struct {
 	requireMapListKeysMapSetValidation bool
 	// preexistingExpressions tracks which CEL expressions existed in an object before an update. May be nil for create.
 	preexistingExpressions preexistingExpressions
+	// mutatedVersions tracks which versions are changed when updating a CRD. Does not apply to creation or deletion.
+	// Some checks use this to avoid rejecting previously accepted versions due to a control plane upgrade/downgrade.
+	mutatedVersions sets.Set[string]
```

Comment:
reversing the polarity of this to `versionsWithUnchangedSchemas` makes the empty value correct for create cases

```suggestion
	versionsWithUnchangedSchemas sets.Set[string]
```